src/main/java/org/mockito/quality/Strictness.java

```diff
@@ -89,5 +94,49 @@
      * @since 2.3.0
      */
     @Incubating
-    STRICT_STUBS;
+    STRICT_STUBS,
+
+    /**
+     * Enforces declaring stubbings for all <strong>non-void</strong> invocations.
+     * Please prefer {@link #STRICT_STUBS} for cleaner tests.
+     * If you are convinced you need {@code STRICT_MOCKS} mocking then read on.
+     * <p>
+     * This mode requires the developer to declare all stubbings for <strong>non-void</strong> methods
+     * invoked in the production code.
+     * Please remember that <strong>void methods</strong> are ignored and you are not required to stub them.
+     * <p>
+     * {@code STRICT_MOCKS} mocking is similar to traditional mocking with EasyMock,
+     * where undeclared invocations would fail early.
+     * {@code STRICT_MOCKS} mocking is useful for users migrating from EasyMock
+     * who are emotionally attached to stricter mocking.
```

Comment:
Let's not use words like `emotionally` here.
```suggestion
     * who prefer the behavior of stricter mocking.
```

Reply:
+1